packages/client/src/returns/types/getReturnWorkflow.types.ts

```diff
@@ -0,0 +1,14 @@
+import type { Config } from '../..';
+import type { Return } from './return.types';
+
+export type ReturnWorkflow = {
+  isCompleted: boolean;
+  sort: number;
+  status: { code: string };
```

Comment:
I think we can have an enum for the status code instead of assuming just **string**.
```suggestion
  status: { code: StatusCodeReturnWorkflows };
```

```
enum StatusCodeReturnWorkflows {
    WAITING_EXTERNAL_APPROVAL = 'WaitingExternalApproval',
    DRAFT = 'Draft',
    BOOKED = 'Booked',
    AT_PARTNER_LOCATION = 'AtPartnerLocation',
    IN_TRANSIT = 'InTransit',
    ARRIVED_AT_FINAL_LOCATION = 'ArrivedAtFinalLocation',
    PROCESSING = 'Processing',
    ACCEPTED = 'Accepted',
    PARTIALLY_ACCEPTED = 'PartiallyAccepted',
    REFUSED = 'Refused',
    CANCELLED = 'Cancelled',
};
```

Reply:
Just to add to tiago's suggestion, yes we should if possible add an enum here and in that case, the name of the enum items should be in pascal case, i.e.:

```js
enum StatusCodeReturnWorkflows {
    WaitingExternalApproval = 'WaitingExternalApproval',
    ...
```